### PR TITLE
Maven enforcer added and maven plugin version properties refactored slightly

### PR DESCRIPTION
**Pull request by jstourac**

### CODE REVIEW

**Comment by sgala** [comment]
Hi, when I checked out and tried to run the integration tests, I get an error as soon as I specify "-Pcomplete" so I guess there is some problem in the change:

```
$ mvn clean verify -Pcomplete
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-failsafe-plugin must be a valid version but is '${version.org.apache.maven.plugins.maven-failsafe-plugin}'. @ line 55, column 26
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.jboss.qa.noe:noe-core-testsuite:0.17.13-SNAPSHOT (/home/sgala/RedHat/Code/noe-core/testsuite/pom.xml) has 1 error
[ERROR]     'build.plugins.plugin.version' for org.apache.maven.plugins:maven-failsafe-plugin must be a valid version but is '${version.org.apache.maven.plugins.maven-failsafe-plugin}'. @ line 55, column 26
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

**Comment by jstourac** [comment]
Oh, you're right. Sorry about that. It should be fixed now. I tried the `mvn clean verify -Pcomplete` locally now both with `master` and with my changes. In both cases it there are test failures which look same. So not sure whether I miss some local configuration on my machine or whether it's expected or something else.

File: testsuite/pom.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
```
Comment (sgala) [inline testsuite/pom.xml:54]: I already commented about the error given running the integration tests by this line. Integration tests pass now, so the PR looks ok to me.

**Review by sgala — APPROVED** [review]

**Comment by jstourac** [comment]
Hi guys, gentle bump on this one :slightly_smiling_face:

**Comment by PaulLodge** [comment]
@sgala Can you test and merge this PR please. 
@jstourac is this PR urgent?

**Review by PaulLodge — APPROVED** [review]

**Pull request merged**